test/test_local_db.js

```diff
@@ -317,25 +292,79 @@ var testparseMongoConnectionString = function (cb) {
 
     async.eachSeries(testGoodArray, function (testString, cb) {
       var compareVal = undefined;
-      console.log("Testing string " + testString);
+
       assert.doesNotThrow(function () {
         compareVal = localdb.parseMongoConnectionURL(testString)
-      });
+      }, `parsing string "${testString}" should be successful`);
+
       assert.ok(compareVal);
+
       cb();
-    }, function () {
-      cb();
+    }, function (e) {
+      cb(e);
     });
   }
 
-  async.series([testBadStrings, testGoodStrings], function () {
+  async.series([testGoodStrings], function () {
     cb();
   });
 };
 
+exports['test errors should be thrown for invalid connection strings'] = function () {
+  var localdb = require("../lib/localdb.js");
+
+  var BAD_STRINGS = {
+    MISSING_AUTH_SECTION: 'mongodb://user2@localhost:27017,localhost:27017/someReplicaApp',
+    NO_DATABASE: 'mongodb://user2:pass2@localhost:27017,localhost:27017',
+    NO_HOST: 'mongodb://user2:pass2@/someReplicaApp',
+    GARBAGE: 'afgadf ga dfg adfg adfg asd ag:;;////',
+    NOTHING: '',
+
+    // TODO - It's probably more worthwhile to check that a required option exists or not in the below tests?
```

Comment:
Ya I think these tests are ok to remove, they're testing the same logic it looks like so +1